src/components/cc-addon-header/cc-addon-header.smart-jenkins.js

```diff
@@ -0,0 +1,107 @@
+import { fakeString } from '../../lib/fake-strings.js';
+import { defineSmartComponent } from '../../lib/smart/define-smart-component.js';
+import '../cc-smart-container/cc-smart-container.js';
+import { CcAddonHeaderClient } from './cc-addon-header.client.js';
+import './cc-addon-header.js';
+// @ts-expect-error FIXME: remove when clever-client exports types
+import { getAddon as getAddonProvider } from '@clevercloud/client/esm/api/v2/providers.js';
+// @ts-expect-error FIXME: remove when clever-client exports types
+import { ONE_SECOND } from '@clevercloud/client/esm/with-cache.js';
+import { sendToApi } from '../../lib/send-to-api.js';
+
+const PROVIDER_ID = 'jenkins';
+
+/**
+ * @typedef {import('./cc-addon-header.js').CcAddonHeader} CcAddonHeader
+ * @typedef {import('./cc-addon-header.types.js').JenkinsProviderInfo} JenkinsProviderInfo
+ * @typedef {import('./cc-addon-header.types.js').RawAddon} RawAddon
+ * @typedef {import('../cc-zone/cc-zone.types.js').ZoneStateLoaded} ZoneStateLoaded
+ * @typedef {import('../../lib/smart/smart-component.types.js').OnContextUpdateArgs<CcAddonHeader>} OnContextUpdateArgs
+ * @typedef {import('../../lib/send-to-api.js').ApiConfig} ApiConfig
+ */
+
+defineSmartComponent({
+  selector: 'cc-addon-header[smart-mode=jenkins]',
```

Comment:
nitpick: there's no double quotes around the value of the smart-mode parameter unlike elsewhere